applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -208,6 +253,12 @@ Otherwise, the image is always uploaded to the slot 1 and then moved to slot 0 b
 .. note::
    Make sure to properly configure the bootloader to ensure that the build system generates the :file:`zephyr/dfu_application.zip` archive containing all of the required update images.
 
+SUIT
+----
+
+The :file:`DFU/root.suit` SUIT envelope is used in the SUIT DFU procedure.
```

Comment:
maybe it would be nice to specify it is located in the build directory?
```
The :file:`<build_dir>/DFU/root.suit` SUIT envelope is used in the SUIT DFU procedure.
```